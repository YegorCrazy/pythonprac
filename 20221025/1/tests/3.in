print(list(fib(5, 8)))
