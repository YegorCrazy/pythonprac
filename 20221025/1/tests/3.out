[8, 13, 21, 34]
